send-transaction-service/src/send_transaction_service.rs

```diff
@@ -18,8 +21,18 @@ use {
 
 /// Maximum size of the transaction queue
 const MAX_TRANSACTION_QUEUE_SIZE: usize = 10_000; // This seems like a lot but maybe it needs to be bigger one day
+
+/// Maximum batch size for sending transaction in batch
+/// When this size is reached, send out the transactions.
+const DEFAULT_TRANSACTION_BATCH_SIZE: usize = 100;
+
 /// Default retry interval
 const DEFAULT_RETRY_RATE_MS: u64 = 2_000;
+
+/// Default maximum batch waiting time in ms. If this time is reached,
+/// whatever transaction cached will be sent.
+const DEFAULT_BATCH_SEND_RATE_MS: u64 = 200;
```

Comment:
```suggestion
const DEFAULT_BATCH_SEND_RATE_MS: u64 = 1;
```
limit to 1000TPS by default here too